### PR TITLE
github support

### PR DESCRIPTION
**Pull request by natebrennand**
Searches the event feed of every user found in an Org and greps for an email field. I think this is easier than trying to put varied data into structs (not all events are the same type).
This gets the majority of them, found 28 for us.

### CODE REVIEW

**Comment by natebrennand** [comment]
cc @bstein-clever, you now have a datasource for directly notifying after force pushes!

File: integrations/github/github.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	gh := githubAPI.NewClient(tc)
+
+	lo := githubAPI.ListMembersOptions{}
+	for {
+		members, resp, err := gh.Organizations.ListMembers(l.Org, &lo)
+		if err != nil {
+			return u, fmt.Errorf("Failed to form HTTP request for Github => {%s}", err)
+		}
+		for _, m := range members {
+			_ = m
```
Comment (xavi-) [inline integrations/github/github.go]: Excuse my ignorance, but what's the point of this assignment?  The variable `m` is already being used down below.

File: integrations/github/github.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+			if m.Login != nil && *m.Login != "" {
+				email := findEmail(gh, *m.Login)
+				if email != "" {
+					user, exists := u[email]
+					if exists {
+						user.Github = *m.Login
+						u[email] = user
+					}
+				}
+			}
```
Comment (xavi-) [inline integrations/github/github.go]: Nit.  Maybe consider inverting logic to and using `continue` to reduce nesting depth?

File: integrations/github/github.go (hunk 1 above)
Comment (natebrennand) [inline integrations/github/github.go]: left over from prototyping phase, will remove

File: integrations/github/github.go (hunk 2 above)
Comment (natebrennand) [inline integrations/github/github.go]: agreed. updated

File: integrations/github/github.go [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+		if e.RawPayload != nil {
+			matches := emailRgx.FindAllStringSubmatch(string(*e.RawPayload), 1)
+			if len(matches) == 1 && len(matches[0]) == 2 {
+				return strings.ToLower(matches[0][1])
+			}
+		}
+	}
+
+	return ""
+}
```
Comment (xavi-) [inline integrations/github/github.go]: Great job with this integration.  It's a great hack and way less insane then I thought it was going to be.

Comment (natebrennand) [inline integrations/github/github.go]: Thanks, this seemed like a saner option than trying to parse the list of events that were of inconsistent structures. And this actually works better than expected.
Only issue is users that have a older short email in their git configs. I'm gonna add a log message for that in order to nag them so they get indexed.

File: integrations/user.go [hunk 4]
```diff
@@ -27,8 +29,8 @@ var (
 	EmailIndex = Index{"", "email"}
 	// FreeTierThroughput is set low within the free tier
 	FreeTierThroughput = schema.ProvisionedThroughput{
-		ReadCapacityUnits:  1,
-		WriteCapacityUnits: 1,
+		ReadCapacityUnits:  2,
+		WriteCapacityUnits: 2,
```
Comment (xavi-) [inline integrations/user.go]: I noticed the same changes in #5 (https://github.com/Clever/who-is-who/pull/5/files#diff-38965569a8d99855484fad39c6d5eae8L30).  Possible merge conflict?

**Comment by xavi-** [comment]
lgtm.

File: integrations/user.go (hunk 4 above)
Comment (natebrennand) [inline integrations/user.go]: it's the same commit, should be fine when I rebase.

**Pull request merged**